live-examples/js-examples/functions/meta.json

```
@@ -1,16 +1,22 @@
 {
-    "pages": {
-        "functionsArguments": {
-            "exampleCode": "./live-examples/js-examples/functions/functions-arguments.js",
-            "fileName": "functions-arguments.html",
-            "title": "Демо JavaScript: Аргументи функцій",
-            "type": "js"
-        },
-        "functionsArrow": {
-            "exampleCode": "./live-examples/js-examples/functions/functions-arrow.js",
-            "fileName": "functions-arrow.html",
-            "title": "Демо JavaScript: Функції =>",
-            "type": "js"
-        }
+  "pages": {
+    "functionsArguments": {
+      "exampleCode": "./live-examples/js-examples/functions/functions-arguments.js",
+      "fileName": "functions-arguments.html",
+      "title": "Демо JavaScript: Аргументи функцій",
+      "type": "js"
+    },
+    "functionsArrow": {
+      "exampleCode": "./live-examples/js-examples/functions/functions-arrow.js",
+      "fileName": "functions-arrow.html",
+      "title": "Демо JavaScript: Функції =>",
+      "type": "js"
+    },
+    "functionsDefault": {
+      "exampleCode": "./live-examples/js-examples/functions/functions-default.js",
+      "fileName": "functions-default.html",
+      "title": "Демо JavaScript: Функції усталено",
```

```suggestion
      "title": "Демо JavaScript: Усталені значення параметрів функцій",
```
Стисло тут не передати